src/sponge.rs

```diff
@@ -18,7 +18,7 @@ pub struct Sponge<F: Field, MP: MultiPermutation<F>> {
     permutation: MP,
     bitrate: usize,
     capacity: usize,
-    phantom: PhantomData<F>,
+    phantom: PhantomData<*const F>,
```

Comment:
`PhantomData<*const F>` is the preferred declaration according to the documentation.